src/shared/utils/now.ts

```diff
@@ -0,0 +1,3 @@
+import { formatISO } from 'date-fns';
+
+export const now = formatISO(Date.now());
```

Comment:
this is wrong. the now function will always be the time of running your application for the first time.

Reply:
I don't understand the requirement well. Will leave it till we discuss it further.